toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -457,6 +468,13 @@ func customizeOSContents(ic *ImageCustomizerParameters) error {
 		}
 	}
 
+	if ic.outputArtifactsDir != "" {
```

Comment:
Move this into `CustomizeImage`, so that it sits beside the other final output code.